aesthetic_ants/penguin/player.py

```diff
@@ -0,0 +1,30 @@
+import pyglet
+import pyglet.window.key as key
+from utils import angle_between, loader, keys
+from math import degrees
+
+
+class Player(pyglet.sprite.Sprite):
+    def __init__(self, x, y):
+        player_image = loader.image("penguin.png")
+
+        # Rotate about the center
+        player_image.anchor_x = player_image.width // 2
+        player_image.anchor_y = player_image.height // 2
+
+        super().__init__(player_image, x=x, y=y)
+
+    def update(self, **kwargs):
+        super().update(**kwargs)
```

Comment:
I forgot to tell you this, apologies, but `update()` is meant to take one argument: `dt` -- the time it took from one update call to the next. This is because this would be called in the main update function which takes one a `dt` argument as well. The `**kwargs` are okay though.

As you might've noticed this also tromps on the Sprite's update so it might be better to rename it to something like `on_update(self, dt)` later on (not right now though).